db/blob/blob_file_cache.cc

```diff
@@ -23,7 +23,8 @@ BlobFileCache::BlobFileCache(Cache* cache,
                              uint32_t column_family_id,
                              HistogramImpl* blob_file_read_hist)
     : cache_(cache),
-      mutex_(kNumberOfMutexStripes, GetSliceNPHash64),
+      mutex_(kNumberOfMutexStripes,
+             static_cast<uint64_t (*)(const Slice&)>(&GetSliceNPHash64)),
```

Comment:
On first inspection, I thought this was UB but it appears legit. https://en.cppreference.com/w/cpp/language/overloaded_address

Consider stuffing in a constant in hash.h, without the static_cast, perhaps `kGetSliceNPHash64UnseededFnPtr`. But OK as is.

Reply:
Hid it as suggested. I also only learned this feature after adding an overload caused a compiler error.